proto/v2/v2.proto

```diff
@@ -19,11 +19,13 @@ package datacommons.v2;
 
 import "entity.proto";
 
-message Arc {
-  // The property of one single arc. This is only a single property, rather than
-  // chained properties
-  string property = 1;
-  repeated datacommons.EntityInfo nodes = 2;
+message Nodes {
+  repeated datacommons.EntityInfo nodes = 1;
+}
+
+message LinkedGraph {
```

Comment:
Change this to LinkedGraph, which is much better.

Reply:
This LG.  Can we describe the keys and what the fields mean, maybe with an example?

Reply:
Also, I forget what the consensus was on using map vs. list of protos (for json purposes), do we use maps in V1?

Reply:
Since V2 holds a subgraph in the response, it's better to use map than list. We use list in V1 and can still provide util functions to convert the raw v2 response into list representations, especially for observations.